packager/hls/base/simple_hls_notifier.cc

```diff
@@ -41,51 +40,51 @@ const char kWidevineDashIfIopUUID[] =
     "urn:uuid:edef8ba9-79d6-4ace-a3c8-27dcd51d21ed";
 
 bool IsWidevineSystemId(const std::vector<uint8_t>& system_id) {
-  return system_id.size() == arraysize(media::kWidevineSystemId) &&
+  return system_id.size() == std::size(media::kWidevineSystemId) &&
          std::equal(system_id.begin(), system_id.end(),
                     media::kWidevineSystemId);
 }
 
 bool IsCommonSystemId(const std::vector<uint8_t>& system_id) {
-  return system_id.size() == arraysize(media::kCommonSystemId) &&
+  return system_id.size() == std::size(media::kCommonSystemId) &&
          std::equal(system_id.begin(), system_id.end(), media::kCommonSystemId);
 }
 
 bool IsFairPlaySystemId(const std::vector<uint8_t>& system_id) {
-  return system_id.size() == arraysize(media::kFairPlaySystemId) &&
+  return system_id.size() == std::size(media::kFairPlaySystemId) &&
          std::equal(system_id.begin(), system_id.end(),
                     media::kFairPlaySystemId);
 }
 
 std::string Base64EncodeData(const std::string& prefix,
                              const std::string& data) {
     std::string data_base64;
-    base::Base64Encode(data, &data_base64);
+    absl::Base64Escape(data, &data_base64);
     return prefix + data_base64;
 }
 
 std::string VectorToString(const std::vector<uint8_t>& v) {
     return std::string(v.begin(), v.end());
 }
 
-// TODO(rkuroiwa): Dedup these with the functions in MpdBuilder.
-// If |media_path| is contained in |parent_path|, then
-//   Strips the common path and keep only the relative part of |media_path|.
-//   e.g. if |parent_path| is /some/parent/ and
-//           |media_path| is /some/parent/abc/child/item.ext,
-//        abc/child/item.ext is returned.
-// else
-//   Returns |media_path|.
-// The path separator of the output is also changed to "/" if it is not.
-std::string MakePathRelative(const std::string& media_path,
-                             const FilePath& parent_path) {
-  FilePath relative_path;
-  const FilePath child_path = FilePath::FromUTF8Unsafe(media_path);
-  const bool is_child =
-      parent_path.AppendRelativePath(child_path, &relative_path);
-  if (!is_child)
-    relative_path = child_path;
-  return relative_path.NormalizePathSeparatorsTo('/').AsUTF8Unsafe();
+// TODO: this is duplicated with MPD module
```

Comment:
this seems like a good time to handle these TODO's and move this out of MPD module somewhere shared we can use in both

Reply:
moved to file_util.h